src/change_stream.ts

```diff
@@ -634,6 +638,7 @@ function waitForTopologyConnected(
     }
 
     if (calculateDurationInMs(start) > timeout) {
+      // TODO: Replace with MongoNetworkTimeoutError
```

Comment:
May want to flag with with a ticket number so we don't lose track of it or just resolve it here